web/react/components/delete_post_modal.jsx

```diff
@@ -149,10 +157,10 @@ export default class DeletePostModal extends React.Component {
                         {'Cancel'}
                     </button>
                     <button
+                        ref='deletePostBtn'
                         type='button'
                         className='btn btn-danger'
                         onClick={this.handleDelete}
-                        autoFocus='autofocus'
```

Comment:
Did you try making this `autoFocus={true}`? Not sure if that would work without using the timeout but it might.